# only build thing we don't provide with EasyBuild
# Ward Poelmans <wpoely86@gmail.com>
diff -ur fsl/extras/build fsl.new/extras/build
--- fsl/extras/build	2016-11-15 15:30:21.000000000 +0100
+++ fsl.new/extras/build	2017-05-19 15:21:07.321630239 +0200
@@ -106,6 +106,8 @@
 fi
 PROJECTS="${PROJECTS} libgd libgdc libprob libcprob newmat cprob newran fftw"
 PROJECTS="${PROJECTS} boost libxml2-2.9.2 libxml++-2.34.0 libsqlite libnlopt ../include/armawrap/dummy_newmat"
+# For EasyBuild:
+PROJECTS="libprob libcprob newmat cprob newran libxml++-2.34.0 libgdc"
 for projname in $PROJECTS; do
     if [ -d $FSLESRCDIR/$projname ] ; then
        buildIt $FSLESRCDIR $projname 1
